routes/personal/personal.controller.js

```diff
@@ -76,12 +75,17 @@ const postConfirmMaritalStatus = (req, res, next) => {
     return res.redirect('/offramp/maritalStatus')
   }
 
+  // Just basing the difference in months or years is not a good enough check so we are checking the total delta days
+  if (getDateDelta(req.session.personal.dateOfBirth) === 23725 && confirmMaritalStatus === 'Yes') {
```

Comment:
This will need to change every day, right?

Reply:
Hey Nick! 

This works if someone finishes the marital status flow, but it doesn't guard against someone just straight up loading the `/deductions/senior-public-transit` URL in the browser. 

To solve this (ie, you shouldn't be able to see the `public-transit` urls at all depending on your date of birth), we should:

- point marital status pages at `public-transit` pages
- add middleware to all of the `public-transit` routes (GET and POST) to redirect to `trillium/rent` if we load this page but the DoB is wrong
- (since our tests will all fail, we can also include the `NODE_ENV !== 'production' ` in there or something) 
-